testing/execution_engine_integration/src/nethermind.rs

```diff
@@ -47,6 +47,12 @@ pub fn build(execution_clients_dir: &Path) {
     build_utils::check_command_output(build_result(&repo_dir), || {
         format!("nethermind build failed using release {last_release}")
     });
+
+    // Cleanup some disk space by removing nethermind's tests
+    let tests_dir = execution_clients_dir.join("nethermind/src/tests");
+    if let Err(e) = fs::remove_dir_all(tests_dir) {
+        eprintln!("Error while deleting folder: {}", e);
+    }
```

Comment:
Wow just realized these are 2.44 GB on my machine, perhaps we don't even need to clone them in the first place? I think we can optionally exclude submodule during checkout:

https://github.com/sigp/lighthouse/blob/aee58b054feafd22e506f21b8512ab62c8ce3928/testing/execution_engine_integration/src/build_utils.rs#L44